### DISCUSSION TITLE
readCsv parametresi

### DISCUSSION THREAD

**Discussion by hsyndnc** [gusanmaz/BMB212-Algorithms-2223#18] [Q&A] [answered]
readCsv fonksiyonuna parametre olarak student_10.csv yolunu vermemiz gerekiyor.Siz derste Student.java ya hiçbir şey yazmıcaksınız demiştiniz.Biz bu işlemi main.java da kendimiz bir student dizisi oluşturarak mı yapmalıyız.Yoksa sizin hazır olarak verdiğiniz student.java nın içindeki readCsv fonksiyonlarına mı parametre vermemiz gerekiyor.

**Comment by gusanmaz** [comment, marked as answer]
Sorunun cevabini surada bulabilirsiniz: https://github.com/gusanmaz/BMB212-Algorithms-22-23/discussions/4#discussioncomment-5434721

Odeviniz https://github.com/gusanmaz/BMB212-Algorithms-22-23/tree/main/data/students bu dizindeki CSV dosyalarina benzer baska dosyalarla test edilebilir. 

`student_10.csv` ozel bir dosya degil. Yazdiginiz kodlarda ozel olarak bu dosyayi okumaya calismaniz size verilen sorunun tam anlasilmadigini gosteriyor. 

**Programiniz okuyacagi ogrenci CSV dosyasinin yolu(pathi) komut satiri argumani olarak programa verilecek!**

Eger hala odevlerde (komut satiri argumani ifadesiyle) ne istendigi netlesmediyse 🔽

* Gecen seneki Odev 4, 5 ve 6'nin aciklamalarini okuyunuz. Gecen senenin ders reposu: https://github.com/gusanmaz/BMB212_Algorithms
* Gecen seneki not tablolarindan hangi kodlarin neden yuksek, hangi kodlarin neden dusuk puan aldigini irdeleyiniz. Ilgili satirda Github handle uzerine gelerek o kullanicinin gondermis oldugu kodlari acilan pop-up penceresinde gorebilirsiniz.
  * Odev Notlandirma Sistemi Hakkinda Detayli Aciklama: https://github.com/gusanmaz/BMB212_Algorithms/blob/main/grades.md
  * Odev 4 Not tablosu: https://gusanmaz.github.io/BMB212_Algorithms/hw4_results.html
  * Odev 5 Not tablosu: https://gusanmaz.github.io/BMB212_Algorithms/hw5_results.html
  * Odev 6 Not tablosu https://gusanmaz.github.io/BMB212_Algorithms/hw6_results.html

Tablolardaki testlerde kullanilan dosyalari 🔽

* list1.txt:  `       `
* list2.txt: `8      `
* list3.txt: `0      `
* list4.txt: `-9 4 5 4 543 32 1`
* list5.txt: `5 6`
* list6.txt: `8 99 -999 90 13 2`
* list7.txt: `10    56  67 90 90`
* list8.txt: `89 30 12 14 16   88`
* list9.txt: `88   888  888 8`
* list10.txt: `12345 907`

Not: Liste icerikleri https://github.com/gusanmaz/BMB212_Algorithms/blob/main/grades.md baglantisindan da gorulebilmektedir.
